packages/image-utils/src/Image.ts

```diff
@@ -73,13 +79,9 @@ async function resizeAsync(imageOptions: ImageOptions): Promise<Buffer> {
 
     if (imageOptions.borderRadius) {
       const mask = Buffer.from(
-        `<svg><rect x="0" y="0" width="${width}" height="${height}" 
-        rx="${imageOptions.borderRadius}" ry="${imageOptions.borderRadius}" 
-        fill="${
-          backgroundColor && backgroundColor !== 'transparent' ? backgroundColor : 'none'
-        }" /></svg>`
+        `<svg><circle cx="${imageOptions.borderRadius}" cy="${imageOptions.borderRadius}" r="${imageOptions.borderRadius}" /></svg>`
       );
-      sharpBuffer.composite([{ input: mask, blend: 'dest-over' }]);
+      sharpBuffer.composite([{ input: mask, blend: 'dest-in' }]);
```

Comment:
with these changes, when you set `foregroundImage` and `backgroundColor`, `ic_launcher_round` is missing the background color

It looks like the call to `.composite()` does away with the background set in the previous composite? Because if you remove this line, the returned image _does_ have the proper background color. 

One way to fix this is to add back the `fill` prop in the svg and then composite twice:
```
sharpBuffer.composite([
        { input: mask, blend: 'dest-over' },
        { input: mask, blend: 'dest-in' },
      ]);
```